src/components/ClientEditDialog.js

```diff
@@ -120,6 +140,22 @@ const ClientEditDialog = (props) => {
                                 </FormHelperText>
                             </Box>
                         </Grid>
+                        <Grid item xs={12} lg={5}>
+                            <Box my={2}>
+                                <Grid component='label' container alignItems='center' spacing={1}>
+                                    <Grid item>Inactive</Grid>
+                                    <Grid item>
+                                        <Switch
+                                            checked={state.checkedA}
+                                            onChange={handleChange}
+                                            name='checkedA'
```

Comment:
what does the `A` stand for in `checkedA`? This name can probably be more descriptive like `activeSelector` or `isActiveCheckBox` or `clientActiveSwitch`